src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/clients/bulkscanprocessor/BulkScanProcessorClientTest.java

```diff
@@ -53,4 +67,102 @@ public void should_throw_exception_when_requested_service_is_not_configured() th
 
     }
 
+    @Test
+    public void should_get_reconciliation_report_with_discrepancy() throws IOException {
+
+        String requestBody = Resources.toString(
+            getResource("reconciliation/statement-request-to-processor.json"),
+            UTF_8
+        );
+
+        String responseBody = Resources.toString(
+            getResource("reconciliation/statement-response-with-discrepancies.json"),
+            UTF_8
+        );
+
+        // given
+        stubFor(post("/reports/reconciliation")
+            .withRequestBody(equalToJson(requestBody))
+            .withHeader("Content-Type", equalTo("application/json"))
+            .withHeader("Accept", equalTo("application/json"))
```

Comment:
do we need `Accept` header?

Reply:
it adds accept header when you add `produces` to feign endpoint in feign client, this verifies it.